### PR TITLE
Cleanup Label Descriptions

### LINKED ISSUE #18147: Cleanup label descriptions
**What should be cleaned up or changed**:
Reconcile SIG and WG names in GitHub label descriptions with doc conventions:
- https://kubernetes.io/docs/contribute/style/style-guide/#kubernetes-io-word-list

e.g. instead of sig-foo, SIG Foo

**Provide any links for context**:
https://github.com/kubernetes/test-infra/pull/18063/files#r444610206

/good-first-issue
/cc @sftim @justaugustus

### PR DESCRIPTION
**Pull request by kurokochin**
This PR will resolve #18147 

In this PR, we rename several descriptions value from sig-* and wg-* patterns to SIG <sig_name> and WG <wg_name> patterns respectively.

### CODE REVIEW

**Comment by cblecker** [comment]
@kurokochin `ERROR: labels.md out of date. Fix with hack/update-labels.sh`

**Comment by kurokochin** [comment]
@cblecker sorry I missed that! Should be good now, thanks for the pointer!

File: label_sync/labels.yaml [hunk 1]
```diff
@@ -426,73 +426,73 @@ default:
...
     - color: d2b48c
-      description: Categorizes an issue or PR as relevant to sig-docs.
+      description: Categorizes an issue or PR as relevant to SIG Docs.
       name: sig/docs
       target: both
       prowPlugin: label
       addedBy: anyone
     - color: d2b48c
-      description: Categorizes an issue or PR as relevant to sig-multicluster.
+      description: Categorizes an issue or PR as relevant to SIG Multicluster.
```
Comment (cblecker) [inline label_sync/labels.yaml:489]: This was originally a mistake it looks like. This should be:
```suggestion
      description: Categorizes an issue or PR as relevant to SIG Instrumentation.
```

Comment (kurokochin) [reply label_sync/labels.yaml]: I've update the description for both occurrence of `SIG Instrumentation` :)

**Comment by kurokochin** [comment]
@cblecker any updates on this?

**Pull request merged**